src/core/util/log.cc

```diff
@@ -64,12 +65,62 @@ namespace kovri
 {
 namespace core
 {
+namespace logging = boost::log;
+namespace expr = boost::log::expressions;
+namespace attrs = boost::log::attributes;
+
+
+std::string SetColor(
+    logging::value_ref<logging::trivial::severity_level,
+    logging::trivial::tag::severity> const& severity, bool color)
+{
+  if (!color)
+  {
+    return "";
+  }
+  else if (severity)
+  {
+    switch (severity.get())
+    {
+      case logging::trivial::fatal:
+      case logging::trivial::error:
+        return "\033[31m"; // Red
+      case logging::trivial::info:
+        return "\033[32m"; // Green
+      case logging::trivial::warning:
+        return "\033[33m"; // Yellow
+      case logging::trivial::debug:
+        return "\033[34m"; // Blue
+      case logging::trivial::trace:
+        return "\033[36m"; // Cyan
```

Comment:
Note: I haven't tested on non-bash yet, but I imagine these escape codes only apply to bash? We should implement `--log-disable-color` for non-bash users if this indeed rings true.

Reply:
https://stackoverflow.com/a/2616912 according to him garbage will show up when colour sequences are not supported.

Reply:
Added --log-disable-color